source/Sashimi.AzureServiceFabric/Endpoints/AzureServiceFabricClusterDeploymentTargetTypeProvider.cs

```diff
@@ -49,22 +46,24 @@ public IEnumerable<(string key, object value)> GetFeatureUsage(IEndpointMetricCo
             }
         }
 
+        public ICreateTargetServiceMessageHandler? CreateTargetServiceMessageHandler { get; } = new AzureServiceFabricClusterServiceMessageHandler();
```

Comment:
Curious - why do we new up the handler for SF but inject for CloudService and WebApp